src/sentry/grouping/parameterization.py

```diff
@@ -240,11 +240,11 @@ def num_tokens_from_string(token_str: str) -> int:
     def is_probably_uniq_id(token_str: str) -> bool:
         token_str = token_str.strip("\"'[]{}():;")
         if len(token_str) < _UniqueId.TOKEN_LENGTH_MINIMUM:
-            return False
-        if (
-            token_str[0] == "<" and token_str[-1] == ">"
-        ):  # Don't replace already-parameterized tokens
-            return False
+            return False  # Don't replace short tokens
+        if token_str[0] == "<" and token_str[-1] == ">":
+            return False  # Don't replace already-parameterized tokens
+        if "%s" in token_str:
```

Comment:
Is there a reason not to add `%d` in here?

Reply:
Mostly a combination of the difficulty in testing this and I want to limit the amount of additional work we have do do on each `token_str` until we have a better way to do better performance testing as well